src/components/shared/basicGrid/styles/basicGrid.scss

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.basic-grid-container {
+  .basic-row {
+    display: flex;
+    flex-direction: row;
+
+    .basic-cell-header {
+      font-family: $fontSelawikBold;
```

Comment:
The grid doesn't need to have styles like this if it is a generic grid. The component that uses it should set any font sizes, etc.